app/main.py

```diff
@@ -2,0 +1,37 @@
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
```

Comment:
Code Style issue: Use annotation, it is a good practice. You have done it correctly with `__init__` method parameters and return types, keep it up!